pkg/cmd/step/create/step_create_task.go

```diff
@@ -610,13 +610,14 @@ func (o *StepCreateTaskOptions) createPipelineTaskParams() []pipelineapi.Param {
 func (o *StepCreateTaskOptions) setBuildValues() error {
 	labels := map[string]string{}
 	if o.GitInfo != nil {
-		labels["owner"] = o.GitInfo.Organisation
-		labels["repo"] = o.GitInfo.Name
+		labels[tekton.LabelOwner] = o.GitInfo.Organisation
+		labels[tekton.LabelRepo] = o.GitInfo.Name
 	}
-	labels["branch"] = o.Branch
+	labels[tekton.LabelBranch] = o.Branch
```

Comment:
Will this break anything in `activity.go`? I'm not sure if these labels make their way to `PipelineActivity` (if that is even what activity.go is about) or if the labels on `PipelineActivity` are set by something else. Right now that code uses `v1.LabelBranch` ("branch") in a few places ([example](https://github.com/jenkins-x/jx/blob/f4eaaee84a0b4c2a5497a50a203008879a3f1310/pkg/kube/activity.go#L318)). I think all of that code is unrelated to and unaffected by this change, but mentioning it just in case.

Reply:
Argh, actually, I think this might. Putting it on hold to dig into tomorrow.